test/test_jit.py

```diff
@@ -7567,7 +7567,7 @@ def test_type_call_in_script(self):
         def fn(x):
             return type(x)
 
-        with self.assertRaisesRegex(RuntimeError, "value of type type"):
+        with self.assertRaisesRegex(RuntimeError, "value of type _TensorMeta"):
```

Comment:
I'm not sure to understand what this means? Is it that `type(type(my_tensor))` is not `_TensorMeta` instead of `type`?

Reply:
The error here is raised by TorchScript where it complains that it doesn't know how to deal with values like "torch.Tensor" (which, prior to this PR, was a "type"). After this, it's a "_TensorMeta", so the error message wobbles (but not in a substantive way)

Reply:
Ok, should this be marked as BC-breaking then?

Reply:
We can conservatively mark it as BC breaking but I currently don't think anyone will actually be broken by it in practice.

Reply:
It mostly depends if we want this to appear in the release notes and if so how.
I feel like it is nice to mention it at least no?

Reply:
Yeah, talking about it in internals sgtm